git-hub

```diff
@@ -1291,15 +1291,14 @@ class IssueCmd (CmdGroup):
             # through issues to allow changing labels, assignee and
             # milestone (even when GitHub itself doesn't support it
             # :D)
-            url = '/repos/%s/issues/%s' % (config.upstream, args.issue)
```

Comment:
The comment above might also need to be amended as the `url` is no longer defined here

Reply:
Yeah, good catch!

Reply:
Fixed